L1Trigger/L1THGCal/test/testHGCalL1TGeometryV9Imp2_Nose_cfg.py

```diff
@@ -0,0 +1,155 @@
+    dataset = cms.untracked.PSet(
+        filterName = cms.untracked.string(''),
+        dataTier = cms.untracked.string('GEN-SIM-DIGI-RAW')
+    ),
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('generation_step')
+    )
+)
+
+##################
+
+process.FEVTDEBUGoutput = cms.OutputModule("PoolOutputModule",
+    splitLevel = cms.untracked.int32(0),
+    eventAutoFlushCompressedSize = cms.untracked.int32(5242880),
+    outputCommands = cms.untracked.vstring(
+        'keep *_genParticles_*_*',
+        'keep *_hgcalBackEndLayer1Producer_*_*',
+        'keep *_hgcalBackEndLayer2Producer_*_*',
+        'keep *_hgcalTowerProducer_*_*',
+    ),
+    fileName = cms.untracked.string('file:/tmp/dalfonso/test.root')
+)
+
+##################
+
+# Additional output definition
+process.TFileService = cms.Service(
+    "TFileService",
+    fileName = cms.string("/tmp/dalfonso/test_triggergeom.root")
+    )
+
+
+# Other statements
+process.genstepfilter.triggerConditions=cms.vstring("generation_step")
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
+
+
+process.generator = cms.EDFilter("Pythia8PtGun",
+    PGunParameters = cms.PSet(
+        AddAntiParticle = cms.bool(True),
+        MinEta = cms.double(3.499),
+        MaxEta = cms.double(3.501),
+        MaxPhi = cms.double(3.14159265359),
+        MinPhi = cms.double(-3.14159265359),
+        MinPt = cms.double(19.999),
+        MaxPt = cms.double(20.001),
+# 13 muon and 11 ele works, pions 211 no
+#        ParticleID = cms.vint32(211)
+        ParticleID = cms.vint32(22)
+    ),
+    PythiaParameters = cms.PSet(
+        parameterSets = cms.vstring()
+    ),
+    Verbosity = cms.untracked.int32(0),
+    firstRun = cms.untracked.uint32(1),
+    psethack = cms.string('single pion pt 10 eta 35')
+)
+
+process.mix.digitizers = cms.PSet(process.theDigitizersValid)
+
+
+# Path and EndPath definitions
+process.generation_step = cms.Path(process.pgen)
+process.simulation_step = cms.Path(process.psim)
+process.genfiltersummary_step = cms.EndPath(process.genFilterSummary)
+process.digitisation_step = cms.Path(process.pdigi_valid)
+process.L1simulation_step = cms.Path(process.SimL1Emulator)
+process.digi2raw_step = cms.Path(process.DigiToRaw)
+process.endjob_step = cms.EndPath(process.endOfProcess)
+#process.FEVTDEBUGoutput_step = cms.EndPath(process.FEVTDEBUGoutputGEO)
+process.FEVTDEBUGoutput_step = cms.EndPath(process.FEVTDEBUGoutput)
+
+process.load('L1Trigger.L1THGCal.hgcalTriggerPrimitives_cff')
+# Eventually modify default geometry parameters
+from L1Trigger.L1THGCal.customTriggerGeometry import custom_geometry_V9
+process = custom_geometry_V9(process, 2)
+
+process.hgcaltriggergeomtester = cms.EDAnalyzer(
+    "HGCalTriggerGeomTesterV9Imp2"
+    )
+process.test_step = cms.Path(process.hgcaltriggergeomtester)
```

Comment:
It doesn't seem to be used. So this file doesn't seem to be testing the geometry. So whether the file name should be changed or the geometry testing modules should be used

Reply:
Could it be separated in 2 different configs (it is currently done like that for the other test files)? There are test files dedicated to purely geometry testing and test files dedicated to test the TPG chain.

Reply:
for me its' just me, for this PR I prefer have a single cfg.py to test both. Once 11_0 is out we can expand (i.e. add geometry V1, add multiple cfg, add dedicated DQM for jenkin tests ...)

Reply:
Ok for now